charter/charter-detecting-unwanted-location-trackers.md

```diff
@@ -61,3 +61,4 @@ Since most of the existing tracking accessories use Bluetooth, the DULT WG will
 
 * Submit an informational document about the state of tracker accessory platforms and how they work for publication
 * Submit a standards document defining the protocol to detect and interact with unwanted tracker accessories for publication
+* Produce evidence of efficacy of proposed solution
```

Comment:
I think this doesn't belong as its own milestone. Instead, you'd want to say somewhere that the standards document needs to include a good threat analysis and security analysis before publishing.